From 6b1123b313e439d8bfae3570a3e541b3aa26f3ad Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 16 Sep 2023 19:52:56 -0700
Subject: [PATCH] cppignore.lnx: Ignore PKCS12 and testLaunch test
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

These tests fail on Linux

Upstream-Status: Inappropriate [OE specific]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
Signed-off-by: Patrick Wicki <patrick.wicki@siemens.com>
Signed-off-by: Peter Marko <peter.marko@siemens.com>

Updated comment and content after fixing some tests.
Left only MongoDBTest.testArray on ignore list, untested.

---
 cppignore.lnx | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/cppignore.lnx b/cppignore.lnx
index b3288d474..9aad48ca5 100644
--- a/cppignore.lnx
+++ b/cppignore.lnx
@@ -30,3 +30,4 @@ CppUnit::TestCaller<HTTPSClientSessionTest>.testProxy
 CppUnit::TestCaller<HTTPSStreamFactoryTest>.testProxy
 CppUnit::TestCaller<DNSTest>.testHostByAddress
 CppUnit::TestCaller<DNSTest>.testHostByName
+CppUnit::TestCaller<MongoDBTest>.testArray
